staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -56,6 +56,8 @@ const (
 	ProfileNetadmin = "netadmin"
 	// ProfileSysadmin offers elevated privileges for debugging.
 	ProfileSysadmin = "sysadmin"
+	// ProfileWindows offers elevated privileges for debugging.
+	ProfileWindows = "windows"
```

Comment:
My only question will be, is this going to be the only one windows profile? IIUC the only thing you're setting is the host network, so maybe something like `win-netadmin` or something that will allow us to add additional windows profiles in the future?

Reply:
From today's sig-cli call @brianpursley pointed that `win-sysadmin` might be a better name. Still to be confirmed with sig-windows TLs.

Reply:
+1 for `win-sysadmin` or `winadmin`